### PR TITLE
Dividers

### PR DESCRIPTION
**Pull request by ryanmcdonough**
Inclusion of dividers according to the Material Design spec with multiple DPI pixel heights.

### CODE REVIEW

File: dist/css/material-wfont.css [hunk 1]
```diff
@@ -30890,6 +30896,47 @@ icon-material-blue-grey-A700 {
 .panel-footer {
   background-color: #ececec;
 }
+hr.on-dark {
+  color: #1a1a1a;
+}
+hr.on-light {
+  color: #ffffff;
+}
+@media (-webkit-min-device-pixel-ratio: 0.75), (min-resolution: 120dpi) {
```
Comment (FezVrasta) [inline dist/css/material-wfont.css:30905]: this prevents compatibility with not webkit/blink browsers. Probably removing the vendor prefix will allow autoprefixer to adds the supported prefixes. Not sure tough, you may try.

Comment (ryanmcdonough) [inline dist/css/material-wfont.css:30905]: I added the (min-resolution: *-dpi) as well for the none webkit browsers. I'll try a build without the prefix see if that adds any extra options.

Comment (ryanmcdonough) [inline dist/css/material-wfont.css:30905]: Seems autoprefixer doesn't support pixel density yet, targeted for 5.1 - I'll add in all the vendor prefixes.

**Comment by ryanmcdonough** [comment]
Will update the vendor prefixes and resubmit.

**Pull request closed without merging**